src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -81,7 +81,7 @@
     // can only be returned by the doFinal(...) call.
     private static final int MAX_BUF_SIZE = Integer.MAX_VALUE;
     // data size when buffer is divided up to aid in intrinsics
-    private static final int TRIGGERLEN = 65536;  // 64k
+    private static final int TRIGGERLEN = 4096;  // 64k
```

Comment:
nit: comment should be 4k?

Reply:
Ack.. I changed it to test something and forgot to change it back.. I'll put it back to 64k with the merge-conflict

Reply:
Sure.